.gitignore

```diff
@@ -39,3 +39,8 @@ nb-configuration.xml
 
 # Local environment
 .env
+
+# Elastic Beanstalk Files
+.elasticbeanstalk/*
+!.elasticbeanstalk/*.cfg.yml
+!.elasticbeanstalk/*.global.yml
```

Comment:
That's the defaults which EB injects in the .gitignore file when running `eb init`. I assume those files might be useful in some cases to commit, but we don't have any files matching that pattern currently.

Reply:
Ahh ok... I was just thinking that we don't have these in the Curation .gitignore.... maybe we should add them there also?

Reply:
Ya, I want to wait for the deployment through GH actions to succeed first (to ensure those files are not needed anywhere for deployment) but think it would indeed be a good thing to add to the curation repo after that.